examples/js/plugins/FeatureToolTip.js

```diff
@@ -1,150 +1,224 @@
+            tooltip.style.top = view.eventToViewCoords(event).y + 'px';
+        }
+    }
+
+    function fillToolTip(features, layer, options) {
+        var content = '';
+        var feature;
+        var geometry;
+        var style;
+        var fill;
+        var stroke;
+        var symb = '';
+        var prop;
+
+        for (var p = 0; p < features.length; p++) {
+            feature = features[p];
+
+            geometry = feature.geometry;
+            style = (layer.style && layer.style.isStyle) ? layer.style : geometry.properties.style;
+            fill = style.fill.color;
+            stroke = '1.25px ' + style.stroke.color;
+
+            if (feature.type === itowns.FEATURE_TYPES.POLYGON) {
+                symb = '&#9724';
+            } else if (feature.type === itowns.FEATURE_TYPES.LINE) {
+                symb = '&#9473';
+                fill = style.stroke.color;
+                stroke = '0px';
+            } else if (feature.type === itowns.FEATURE_TYPES.POINT) {
+                symb = '&#9679';
+            }
 
-                    if (feature.properties && !opts.filterAll) {
-                        if (opts.format) {
-                            for (prop in feature.properties) {
-                                if (!opts.filter.includes(prop) && prop != 'style' && prop != 'name' && prop != 'nom' && prop != 'description') {
-                                    opts.format(prop, feature.properties[prop]);
-                                }
-                            }
-                        } else {
-                            content += '<ul>';
-                            for (prop in feature.properties) {
-                                if (!opts.filter.includes(prop) && prop != 'style' && prop != 'name' && prop != 'nom' && prop != 'description') {
-                                    content += '<li>' + prop + ': ' + feature.properties[prop] + '</li>';
-                                }
-                            }
-
-                            if (content.endsWith('<ul>')) {
-                                content = content.replace('<ul>', '');
-                            } else {
-                                content += '</ul>';
-                            }
+            content += '<div>';
+            content += '<span style="color: ' + fill + '; -webkit-text-stroke: ' + stroke + '">';
+            content += symb + ' ';
+            content += '</span>';
+            content += (geometry.properties.name || geometry.properties.nom || geometry.properties.description || layer.name || '');
+
+            if (feature.type === itowns.FEATURE_TYPES.POINT) {
+                content += '<br/><span class="coord">long ' + feature.coordinates[0].toFixed(4) + '</span>';
+                content += '<br/><span class="coord">lat ' + feature.coordinates[1].toFixed(4) + '</span>';
+            }
+
+            if (geometry.properties && !options.filterAllProperties) {
```

Comment:
`filterAllProperties` could explain this option?

Reply:
This is the previous `filterAll` option (see the documentation below)

Reply:
yes, the documentation doesn't give a real explanation.

Reply:
Yes, updated